### DISCUSSION TITLE
Help with binding TS0002 to TS0042

### DISCUSSION THREAD

**Discussion by mzealey** [Koenkk/zigbee2mqtt#11018] [Devices]
I have a wall switch which I want to bind with a battery switch so that the controls on the battery switch turn the wall switch on/off.

Wall switch is detected as TS0002, battery switch detected as TS0042. They have both paired successfully. But there don't seem to be any clusters which I can bind either via the gui or mqtt directly - they both only show getOta and genTime output clusters. Controller is a standard https://sonoff.tech/product/diy-smart-switch/sonoff-dongle-plus/ stock firmware.

The TS2 is showing the following when i press it:

Info MQTT publish: topic 'zigbee2mqtt/switch', payload '{"action":"1_single","battery":null,"linkquality":93}'
Info MQTT publish: topic 'zigbee2mqtt/switch', payload '{"action":"","battery":null,"linkquality":93}'

Wall switch shows the following when I press it:

Info MQTT publish: topic 'zigbee2mqtt/wall switch', payload '{"linkquality":126,"state_l1":"ON","state_l2":"OFF"}'
Info MQTT publish: topic 'zigbee2mqtt/wall switch', payload '{"linkquality":126,"state_l1":"OFF","state_l2":"OFF"}'

So I'm wondering how to bind a given action (ie 1_single) with a given state toggle on the wall switch?

If possible I'd really like to get this working within the zigbee network rather than having to rely on having mqtt around to handle the connections.

**Comment by jirutka** [comment]
I have almost the same problem, also cannot figure out how this is supposed to work.
